README.md

```diff
@@ -1,35 +0,0 @@
-Arteria delivery
-=================
-
-A self contained (Tornado) REST service that checks performs deliveries
```

Comment:
Agreed. I will write up something better here once we know more details about the actual delivery details.